bin/improver-wind-downscaling

```diff
@@ -130,6 +131,12 @@ def main():
                 z0_cube=veg_roughness_cube,
                 height_levels_cube=height_levels).process(wind_speed_slice))
         wind_speed_list.append(result)
+
+    # Temporary fix for chunking problems when merging cubes
+    max_npoints = max([np.prod(cube.data.shape) for cube in wind_speed_list])
+    while iris._lazy_data._MAX_CHUNK_SIZE < max_npoints:
+        iris._lazy_data._MAX_CHUNK_SIZE *= 2
+
```

Comment:
This would have the same effect, but slightly less clear:

if iris._lazy_data._MAX_CHUNK_SIZE < max_npoints:
     iris._lazy_data._MAX_CHUNK_SIZE = int(math.pow(2, math.ceil(math.log(max_npoints, 2))))

Reply:
I have to say I like your first version as-is